### PR TITLE
Store the backup corpus using a signed url

### PR DESCRIPTION
**Pull request by alhijazi**

### CODE REVIEW

File: src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py [hunk 1]
```diff
@@ -1047,6 +1043,30 @@ def handle_corpus_pruning_failures(output: uworker_msg_pb2.Output):
   data_handler.update_task_status(task_name, data_types.TaskState.ERROR)
 
 
+def _fill_backup_urls(fuzz_target: data_types.FuzzTarget,
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py:1046]: Can we pick a better name than "fills" maybe "create"?

**Review by jonathanmetzman — COMMENTED** [review]
lgtm

**Comment by jonathanmetzman** [comment]
When my nit is fixed, feel free to merge this, make corpus_pruning an unprivileged task (batch.py) and then deploy. Also try to kick off the schedule_corpus_pruning (Run now button on https://pantheon.corp.google.com/kubernetes/cronjob/us-central1/clusterfuzz-cronjobs-gke/default/schedule-corpus-pruning/details?project=google.com:clusterfuzz)

File: src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py (hunk 1 above)
Comment (alhijazi) [reply src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py]: Done

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**